pkg/module/go_get_fetcher.go

```diff
@@ -0,0 +1,169 @@
+package module
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/afero"
+)
+
+const (
+	tmpRepoDir = "%s-%s" // owner-repo-ref
```

Comment:
I think this is probably fine as is, but at first I wasn't sure it the comment was meant to reference three parts of the string. If you agree it's a bit confusing, maybe something like `owner-ref`, `owner-version` etc

Reply:
@robjloranger I'll abstract this into a function so that it's much easier to read and more type-safe